### ISSUE TITLE
link to graph_def file in TF Slim models readme.md

### ISSUE THREAD

**Issue by ronrest** [tensorflow/models#2400]
## Feature Request Description
The readme.md file for the TF Slim pre-trained models contains several columns, with information and links.

However, it is missing a column with links to the graph_def file. 

## Proposed solution
I propose something like this (only a subset of the table shown below): 

Model | Code | Graph File | Checkpoint | Top-1 Accuracy| Top-5 Accuracy |
:----:|:------------:|:----------:|:----------:|:-------------:|:--------------:|
..|..|..|..|..|..|
[MobileNet_v1_1.0_224](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[graph](http://download.tensorflow.org/models/mobilenet_v1_1.0_224_frozen.tgz)|[mobilenet_v1_1.0_224_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_1.0_224_2017_06_14.tar.gz)|70.7|89.5|
[MobileNet_v1_0.50_160](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[graph](http://download.tensorflow.org/models/mobilenet_v1_0.50_160_frozen.tgz)|[mobilenet_v1_0.50_160_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_0.50_160_2017_06_14.tar.gz)|59.9|82.5|
[MobileNet_v1_0.25_128](https://arxiv.org/pdf/1704.04861.pdf)|[Code](https://github.com/tensorflow/models/blob/master/slim/nets/mobilenet_v1.py)|[graph](http://download.tensorflow.org/models/mobilenet_v1_0.25_128_frozen.tgz)|[mobilenet_v1_0.25_128_2017_06_14.tar.gz](http://download.tensorflow.org/models/mobilenet_v1_0.25_128_2017_06_14.tar.gz)|41.3|66.2|

I am willing to create a Pull Request with this proposed change.

**Comment by ApolllonDev** [comment]
Hi, may I work on this issue?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
